### PR TITLE
Add API for getting the TAE context active resource

### PR DESCRIPTION
**Pull request by prkalle**
### What this PR does / why we need it
This PR adds runtime API GetTAEContextActiveResource() to get the TAE active resource

### Which issue(s) this PR fixes

Fixes #

### Describe testing done for PR

Unit test and CI passed

### Release note

```release-note
Add API for getting the TAE context active resource
```

### CODE REVIEW

**Review by anujc25 — APPROVED** [review]
LGTM. Thanks

File: tae/tae.go [hunk 1]
```diff
@@ -33,6 +33,16 @@ const (
 	customCommandName string = "_custom_command"
 )
 
+// ResourceInfo resource information of the application engine
+type ResourceInfo struct {
+	// OrgID  ID of the Org
```
Comment (vuil) [inline tae/tae.go:38]: nit: extra space, also "ID of the Organization", and for consistency in case, use Project and Space in the next 2 comments

**Review by vuil — APPROVED** [review]
lgtm, just some nits

File: tae/tae.go (hunk 1 above)
Comment (prkalle) [reply tae/tae.go]: Updated. Thanks

**Pull request merged**